### DISCUSSION TITLE
Idea/Draft PR: Peers list badges

### DISCUSSION THREAD

**Discussion by melyux** [openscopeproject/TrguiNG#39] [Ideas]
I really liked the badges used in the Files tab and wanted to see if they could spruce up the Peers tab. No discussion section in this repo, so made an issue.

I tried it on some candidate columns that seemed to be natural candidates (those with only 2 or 3 possible values). Not sure if you had a philosophy behind using badges and this tramples on it, but it's a nice way to differentiate between these few values by color. Let me know what you think.

![image](https://github.com/openscopeproject/TrguiNG/assets/10296053/f63be7e8-d71d-475b-8900-2bd5344c078e)

**Comment by qu1ck** [comment]
My philosophy here is to add color where it's needed and avoid it otherwise. For example it's important to be able to see priority of a torrent/file clearly as it's an often used feature. Never have I ever cared about the source/protocol/connection type of a peer so having colors there is not useful. It's an interesting bit of trivia to look at once and say "huh, never thought PEX actually works that well" and then never look at it again.

Too much color can be a bad thing too, making the UI a mess. Rapidly updating colors as in peers table is even worse because it is distracting.

Also these things come at a low but non-zero performance cost.

**Reply by qu1ck** [reply]
Moved to discussions.

**Comment by qu1ck** [comment]
I just added "Interface" settings panel. If you make the feature you are proposing an option that you can turn on there I would merge it.
